onebusaway-android/src/main/java/org/onebusaway/android/report/ui/ReportTypeListFragment.java

```diff
@@ -0,0 +1,162 @@
+        }
+
+        Boolean isEmailDefined = isEmailDefined();
+        List<ReportTypeItem> reportTypeItems = new ArrayList<>();
+        for (int i = 0; i < reportTypes.length; i++) {
+            //Don't show the send app feedback section if email is not defined for region
+            if (!isEmailDefined && getString(R.string.rt_app_feedback).equals(reportTypes[i]))
+                continue;
+            ReportTypeItem item = new ReportTypeItem(reportTypes[i], reportDesc[i], reportIcons.getResourceId(i, -1));
+            reportTypeItems.add(item);
+        }
+
+        ReportTypeListAdapter adapter = new ReportTypeListAdapter(getActivity(), reportTypeItems);
+        setListAdapter(adapter);
+        getListView().setOnItemClickListener(this);
+    }
+
+    @Override
+    public void onStart() {
+        super.onStart();
+        ObaAnalytics.reportFragmentStart(this);
+    }
+
+    private Boolean isEmailDefined() {
+        ObaRegion region = Application.get().getCurrentRegion();
+        return !(region == null || region.getContactEmail() == null);
+    }
+
+    @Override
+    public void onItemClick(AdapterView<?> adapterView, View view, int i, long l) {
+        ReportTypeItem rti = (ReportTypeItem) getListView().getItemAtPosition(i);
+        if (getString(R.string.rt_infrastructure_problem).equals(rti.getTitle())) {
+            ((ReportActivity) getActivity()).createInfrastructureIssueActivity();
+
+            ObaAnalytics.reportEventWithCategory(ObaAnalytics.ObaEventCategory.UI_ACTION.toString(),
+                    getString(R.string.analytics_action_problem),
+                    getString(R.string.analytics_label_stop_problem));
+        } else if (getString(R.string.rt_stop_problem).equals(rti.getTitle())) {
+            ((ReportActivity) getActivity()).createInfrastructureIssueActivity(
+                    getString(R.string.ri_selected_service_stop));
+
+            ObaAnalytics.reportEventWithCategory(ObaAnalytics.ObaEventCategory.UI_ACTION.toString(),
+                    getString(R.string.analytics_action_problem),
+                    getString(R.string.analytics_label_stop_problem));
+        } else if (getString(R.string.rt_arrival_problem).equals(rti.getTitle())) {
+            // Report bus stop issue
+            ((ReportActivity) getActivity()).createInfrastructureIssueActivity(
+                    getString(R.string.ri_selected_service_trip));
+
+            ObaAnalytics.reportEventWithCategory(ObaAnalytics.ObaEventCategory.UI_ACTION.toString(),
+                    getString(R.string.analytics_action_problem),
+                    getString(R.string.analytics_label_trip_problem));
+        } else if (getString(R.string.rt_app_feedback).equals(rti.getTitle())) {
+            // Send App feedback
+            String email = getString(R.string.ri_app_feedback_email);
+            UIUtils.sendEmail(getActivity(), email, locationString);
```

Comment:
We just talked about this, but I'll add a comment here too just so its official :) - it looks like this location is always null.  I believe we should be grabbing the location from the Application class here - checkout the current "Help->Contact Us" process for getting location, and we can just imitate that here.